File: frontend/src/app/Business/Models/romaneioDocumento.ts
import { Documento1 } from "./documento1";
import { Romaneio } from "./romaneio";


export interface RomaneioDocumento  {
    id: string;
    romaneioId: string;
    documentoId: string;
    idNavigation: Documento1;
    romaneio: Romaneio;
}

File: backend/Entities/Models/RomaneioDocumento.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class RomaneioDocumento
{
    public Guid Id { get; set; }

    public Guid RomaneioId { get; set; }

    public Guid DocumentoId { get; set; }

    public virtual Documento1 IdNavigation { get; set; } = null!;

    public virtual Romaneio Romaneio { get; set; } = null!;
}
